### PR TITLE
Add Vis and UI color variables

### PR DESCRIPTION
**Pull request by dvdanielamoitzi**
* add visualization and ui colors as scss variables according to the styleguide
* export all scss variables as css variables
* add js function to read css variables to use them in js code

### CODE REVIEW

File: src/scss/main.scss [hunk 1]
```diff
@@ -1,7 +1,10 @@
 /**
  * Created by Samuel Gratzl on 27.08.2014.
  */
+ @import "./abstracts/functions";
```
Comment (thinkh) [inline src/scss/main.scss:4]:
```suggestion
@import "./abstracts/functions";
```

File: src/functions/getCssValue.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
```
Comment (thinkh) [inline src/functions/getCssValue.ts:1]: I suggest moving this file to a `utils` directory as we do it in other repos. What do you think?

**Review by thinkh — APPROVED** [review]
Thanks for putting this together. 👍  In general it looks good to me. I have only suggestion that we could follow if you agree.

**Review by oltionchampari — COMMENTED** [review]
The dist files are not committed

File: src/functions/getCssValue.ts (hunk 2 above)
Comment (thinkh) [reply src/functions/getCssValue.ts]: We agreed in the core meeting to rename it.

**Pull request merged**